apps/projects/app/components/Panel/NewProject/RepoSelector.js

```diff
@@ -265,6 +275,19 @@ const Loading = () => (
     Loading repos...
   </div>
 )
+const NoMoreRepos = () => (
+  <div
+    style={{
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      height: '20%',
+      flexDirection: 'column'
+    }}
+  >
+    <Text>No more repos to add...</Text>
```

Comment:
Maybe just an opinion, but for the sake of consistency we should use same denomination like "repositories" at L108, or change that if we decide to use "repos"

Reply:
Repositories. Changed this line, consistency is important (and looks better).